Add account linking documentation (#73)
terramate-io/terramate-docs: cloud/profile/account-linking.md

---
title: Account Linking Section | Terramate Cloud
description: Learn how to link your Google, GitHub, GitLab and Microsoft accounts with your Terramate Cloud account to sync data between the different platforms.
---

# Account Linking

> Some Terramate Cloud features like [Pull Requests Previews](../previews/index.md), [Deployment Insights](../deployments/index.md),
> [Drift Detection](../drift/index.md) and [Alerts](../alerts/index.md) require you to link your GitLab, GitHub, Microsoft
> and Slack accounts to function properly.
> 
> For example, for Terramate to understand what Pull Requests opened in GitHub are associated with your Terramate Cloud account,
> you need to establish an account mapping between your Terramate Cloud and your GitHub user.

Reviewer: ```suggestion
To utilize features like [Pull Requests Previews](../previews/index.md), [Deployment Insights](../deployments/index.md), [Drift Detection](../drift/index.md), and [Alerts](../alerts/index.md) in Terramate Cloud, you need to link your GitLab, GitHub, Microsoft, and Slack accounts.

For instance, to associate Pull Requests opened in GitHub with your Terramate Cloud account, you must establish an account mapping between your Terramate Cloud and GitHub user.
```

> For that Terramate allows you to connect the following platforms to your Terramate Cloud account:
> 
> - **GitHub:** Links your GitHub to your Terramate Cloud User so that Pull Request Previews, Deployments, and detected
>   Drift for repositories managed on GitHub can be associated with your Terramate Cloud User.
> - **GitLab:** Links your GitLab to your Terramate Cloud User so that Pull Request Previews, Deployments, and detected
>   Drift for repositories managed on GitHub can be associated with your Terramate Cloud User.
> - **Slack (***coming soon***)**: Links your Slack to your Terramate Cloud so that the Slack Bot provided by the
>   [Slack App integration](../integrations/slack.md) can send you direct messages about failed deployments or detected
>   drift that is owned by you.

Reviewer: ```suggestion
Terramate allows you to connect the following platforms to your Terramate Cloud account:

- **GitHub:** Links your GitHub account to your Terramate Cloud user, enabling association with Pull Request Previews, Deployments, and detected drift for repositories managed on GitHub.
- **GitLab:** Links your GitLab account to your Terramate Cloud user, enabling association with Pull Request Previews, Deployments, and detected drift for repositories managed on GitLab.
- **Slack (coming soon):** Links your Slack account to your Terramate Cloud user, allowing the Slack Bot provided by the [Slack App integration](../integrations/slack.md) to send direct messages about failed deployments or detected drift.
```

> Account linking can be done in the [profile section](./index.md) of your Terramate Cloud account. It's recommended to link
> the accounts for all third-party platforms used with Terramate Cloud.

Reviewer: ```suggestion
You can link your accounts in the [profile section](./index.md) of your Terramate Cloud account. We recommend linking all third-party platforms you use with Terramate Cloud.
```
